types/video.js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Video.js 7.2
+// Type definitions for Video.js 8.0
```

Comment:
video.js 8.0 is not out yet, 7.2.2 is the latest.

Reply:
Oh I thought it is the version for @types/video.js, I'll correct it right now.

Btw, since this pr will make a break change, could you tell me how to increase the major version of @types/video.js package?